Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -370,6 +370,14 @@ modes and can be enabled as follows for ``nodePort.mode=hybrid`` in this example
         --set global.k8sServiceHost=API_SERVER_IP \\
         --set global.k8sServicePort=API_SERVER_PORT
 
+In case of a multi-device environment, where Cilium's device auto-detection selects
+more than a single device to expose NodePort, for example, the helm option
+``global.devices={eth0}`` must be additionally specified for the enablement, where
+``eth0`` is the native XDP supported networking device. In that case, the device
+name ``eth0`` must be the same on all Cilium managed nodes. Similarly, the underlying
+driver for ``eth0`` must have native XDP support on all Cilium managed nodes. This
+is typically provided in homogenous cluster environments, for example.
```

Comment:
nit:

```suggestion
driver for ``eth0`` must have native XDP support on all Cilium managed nodes.
```

I don't think the final sentence here actually clarifies anything. A homogenous environment is defined by having the same configuration in all nodes :-)

Reply:
Agree, I'll take it out.